warehouse/query-core/src/main/java/datawave/query/tables/content/ContentQueryTable.java

```diff
@@ -260,4 +277,8 @@ public Set<String> getRequiredQueryParameters() {
     public Set<String> getExampleQueries() {
         return Collections.emptySet();
     }
+
+    public Set<Authorizations> getAuthorizations() {
+        return Collections.singleton(this.scanner.getAuthorizations());
+    }
```

Comment:
Doesn't look like this method is used.  Were you using it for testing at some point?

Reply:
I planned to use this for a test but then did not need it. Will remove.